cmd/cleantool/cleantool.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
done

Reply:
I'm still seeing warnings from `task lint`. Could you please fix all the warnings please? :pray: Sometimes if there are many warnings, it might not show all of it at once, so you may need to run `task lint` multiple times :sweat_smile: . 

Also `task fmt` is a helpful command to fix some formatting. 

FYI, I'm seeing these warnings.
```
$ task lint
task: [lint] bin/golangci-lint run --config=.golangci.yml
task: [lint] bin/golangci-lint run --config=.golangci-new.yml
internal/handlers/common/update.go:971:2: if statements should only be cuddled with assignments (wsl)
        if b == nil {
        ^
internal/handlers/common/update.go:986:2: return statements should not be cuddled if block has more than two lines (wsl)
        return nil
        ^
cmd/cleantool/cleantool.go:15:1: package-comments: should have a package comment (revive)
package main
^
cmd/cleantool/cleantool.go:47:2: only one cuddle assignment allowed before if statement (wsl)
        if token == "" {
        ^
cmd/cleantool/cleantool.go:65:3: only one cuddle assignment allowed before if statement (wsl)
                if err != nil {
                ^
cmd/cleantool/cleantool.go:68:3: only one cuddle assignment allowed before range statement (wsl)
                for _, v := range packages {
                ^
cmd/cleantool/cleantool.go:73:3: if statements should only be cuddled with assignments (wsl)
                if len(packages) < pageSize {
                ^
cmd/cleantool/cleantool.go:61:2: for statement without condition should never be cuddled (wsl)
        for {
        ^
task: Failed to run task "lint": exit status 1
task: Failed to run task "lint": exit status 1
```